### ISSUE TITLE
Volume data display error with the fourth 'STOKES' dimension

### ISSUE THREAD

**Issue by PennyQ** [glue-viz/glue-vispy-viewers#44]
If the data cube is with a fourth dimension as `STOKES` then the 3d-volume-viewer will prompt a error as `Volume visual needs a 3D image.` Is there any smart 'data_loading' scheme in Glue for this problem?

**Comment by astrofrog** [comment]
Yes - open the data from glue and select 'CASA PPV cube' - this needs to be renamed, but will basically split the 4D cube into 4 3D cubes (each will be a component in the dataset)

**Comment by astrofrog** [comment]
Will be solved by https://github.com/glue-viz/glue/issues/924
